index.html

```diff
@@ -66,74 +64,37 @@
     <section class="mdl-article">
       <article class="m-content">
         <header>
-          Firecracker is an open source virtualization technology that is purpose-built for creating and managing
-          secure, multi-tenant container and function-based services.
+          Firecracker is an open source virtualization technology that is purpose-built for creating and managing secure, multi-tenant container and function-based services.
```

Comment:
Since we mostly work this file via code editors (it's not autogenerated), it helps both readability and edit-ability to have multi-paragraph text blocks broken on different lines. My preference is keeping that it as-is (also applies to subsequent of this kind edits below). The current line-break boundary is 120 characters.